releasenotes/notes/0.24/remove-execute_function-max_credits-8c822b8b4b3d84ba.yaml

```
@@ -1,8 +1,15 @@
 ---
-upgrade:
+upgrade_providers:
   - |
-    The ``max_credits`` argument to :func:`~.execute_function.execute`, :func:`~.compiler.assemble` and all
+    The deprecated ``max_credits`` argument to :func:`~.execute_function.execute`, :func:`~.compiler.assemble` and all
     of the ``Qobj`` configurations (e.g. :class:`.QasmQobjConfig` and
-    :class:`.PulseQobjConfig`) has been removed.
-    The credit system has not been in use on IBM Quantum backends for
-    nearly three years, and the option has no effect.  No alternative is necessary.
+    :class:`.PulseQobjConfig`) has been removed. These arguments date back
```

```suggestion
    :class:`.PulseQobjConfig`) has been removed. This argument dates back
```
Please check that this doesn't change meaning.